src/components/promo/projects.tsx

```diff
@@ -0,0 +1,303 @@
+        <div className="w-6 rounded-full border-8 border-[#26254C]" />
+      </div>
+    </div>
+  );
+};
+
+const RedSuitcase = ({ project }: { project: Project }) => {
+  const { title, image, link } = project;
+  return (
+    <div className="flex flex-col hover:[animation-play-state:paused] md:animate-wiggle">
+      <div className="mx-auto -mb-5 h-12 w-20 rounded-md border-8 border-[#8E4C5C]" />
+      <a
+        href={link}
+        target="_blank"
+        rel="noreferrer noopener"
+        className="w-fit min-w-48 rounded-md border-[12px] border-[#C46C81] bg-white transition-all hover:scale-[1.02] hover:shadow-[0px_0px_4px_8px_rgba(196,108,129,0.25)] xl:min-w-64 3xl:min-w-80"
+      >
+        {
+          // eslint-disable-next-line @next/next/no-img-element
+          <img src={image} alt={title} title={title} />
+        }
+      </a>
+    </div>
+  );
+};
+
+const Gear = () => {
+  return (
+    <svg
+      width="130"
+      height="130"
+      viewBox="0 0 130 140"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className="z-50 -my-6 animate-spin-reverse overflow-clip"
+    >
+      <rect
+        x="37"
+        y="44"
+        width="52"
+        height="52"
+        rx="26"
+        fill="#A19CCA"
+        stroke="#777395"
+        strokeWidth="8"
+      />
+      <rect
+        x="47.5"
+        y="54.5"
+        width="31"
+        height="31"
+        rx="15.5"
+        fill="#A19CCA"
+        stroke="#777395"
+        strokeWidth="7"
+      />
+      <path
+        d="M68.8611 48.1279L67.5905 52.9638"
+        stroke="#777395"
+        strokeWidth="5"
+      />
+      <path
+        d="M82.6842 58.4806L79.0204 60.6198"
+        stroke="#777395"
+        strokeWidth="5"
+      />
+      <path
+        d="M50.6404 50.5781L52.9066 53.7584"
+        stroke="#777395"
+        strokeWidth="5"
+      />
+      <path
+        d="M41.7385 64.7822L44.64 65.5446"
+        stroke="#777395"
+        strokeWidth="5"
+      />
+      <path
+        d="M80.9092 75.0742L85.2615 76.2178"
+        stroke="#777395"
+        strokeWidth="5"
+      />
+      <path
+        d="M72.8967 85.8931L75.1629 89.0734"
+        stroke="#777395"
+        strokeWidth="5"
+      />
+      <path
+        d="M57.8315 88.1387L56.688 92.4909"
+        stroke="#777395"
+        strokeWidth="5"
+      />
+      <path
+        d="M47.1396 79.6426L43.9593 81.9088"
+        stroke="#777395"
+        strokeWidth="5"
+      />
+    </svg>
+  );
+};
+
+const listProjects: Project[] = [
+  {
+    image:
+      "https://d112y698adiu2z.cloudfront.net/photos/production/software_thumbnail_photos/002/682/847/datas/medium.png",
+    title: "SuperStage",
+    link: "https://devpost.com/software/superstage",
+  },
+  {
+    image:
+      "https://d112y698adiu2z.cloudfront.net/photos/production/software_photos/002/683/856/datas/medium.png",
+    title: "Infu",
+    link: "https://devpost.com/software/infu",
+  },
```

Comment:
put these in a new file inside /constants/projects.ts

Reply:
fixed